src/myapp.h

```diff
@@ -18,79 +18,77 @@ struct MyApp : wxApp {
     MyApp() : frame(nullptr), serv(nullptr), initateventloop(false) {}
 
     void AddTranslation(const wxString &basepath) {
-        #ifdef __WXGTK__
-            locale.AddCatalogLookupPathPrefix(L"/usr");
-            locale.AddCatalogLookupPathPrefix(L"/usr/local");
-            #ifdef LOCALEDIR
-                locale.AddCatalogLookupPathPrefix(LOCALEDIR);
-            #endif
-            wxString prefix = wxStandardPaths::Get().GetInstallPrefix();
-            locale.AddCatalogLookupPathPrefix(prefix);
-        #endif
+#ifdef __WXGTK__
+        locale.AddCatalogLookupPathPrefix(L"/usr");
+        locale.AddCatalogLookupPathPrefix(L"/usr/local");
+    #ifdef LOCALEDIR
+        locale.AddCatalogLookupPathPrefix(LOCALEDIR);
+    #endif
+        wxString prefix = wxStandardPaths::Get().GetInstallPrefix();
+        locale.AddCatalogLookupPathPrefix(prefix);
+#endif
         locale.AddCatalogLookupPathPrefix(basepath);
         locale.AddCatalog(L"ts", (wxLanguage)locale.GetLanguage());
     }
 
     bool OnInit() {
-        #if wxUSE_UNICODE == 0
-        #error "must use unicode version of wx libs to ensure data integrity of .cts files"
-        #endif
+#if wxUSE_UNICODE == 0
+    #error "must use unicode version of wx libs to ensure data integrity of .cts files"
+#endif
         ASSERT(wxUSE_UNICODE);
 
-        #ifdef __WXMAC__
+#ifdef __WXMAC__

```

Comment:
We definitely want ifdefs indented.. if there are no settings for clang format that can fix this, we either have to fix all ifdefs manually afterwards (like I did last time I think), or not apply clang format.

Reply:
Hi Wouter
Thanks for your feedback. I also did first a automatic round and then picking the right ones with `git add --patch`. Please see the series of commits.
Best regards and a nice sunday
Tobias